src/Manager.php

```diff
@@ -136,6 +136,14 @@ public function isSearchable(string $class): bool
      */
     private function isHandled(HandlerInterface $handler, string $class): bool
     {
-        return \in_array($class, $handler->getHandledClasses(), true);
+        $implements = \class_implements($class);
+        $implements[] = $class;
```

Comment:
To avoid the variable `$implements` starting typed as a `string`, then becoming an `array` it would be better to just do 
```
$implements = [\class_implements($class)];
```

Reply:
\class_implements returns an array?

Reply:
My eyes deceive me, I read it as `class_exists` :unamused: - thanks